controllers/auth.controllers.js

```diff
@@ -21,13 +21,22 @@ const updateUserProfile = async (req, res) => {
   const db = admin.firestore();
   //get user data on request body
   const userId = req.body.userId;
-  const address = req.body.address;
+  const profileImage = req.body.profileImage;
+
+  if (!userId) {
+    res.status(400).send({ message: "userId is required"});
+    return;
+  }
+  // Ensure userId is a string
+  const userIdString = userId.toString();
+
+
   // Add the user's profile to the "example-collection" collection
   db.collection("users")
-    .doc(userId)
+    .doc(userIdString)
     .set({
       userId,
-      address,
+      profileImage,
```

Comment:
can you add image conditions only image can be uploaded, and size should be less than 2 mb
@mukeshsin